sandbox/grist/functions/info.py

```diff
@@ -653,20 +655,72 @@ def is_error(value):
       or (isinstance(value, float) and math.isnan(value)))
 
 
+def _replicate_requests_body_args(data=None, json=None):
+  """
+  Replicate some of the behaviour of requests.post, specifically the data and 
+  json args.
+
+  From testing manually with requests 2.28.2, data overrides json if both supplied.
```

Comment:
Unless there's a very good reason for requests to behave like this, I don't think we should imitate it. Supplying both `data` and `json` suggests a mistake and should probably just raise an error. We can safely change our mind later if we want to.